2024-06-04 18:00:48,379 INFO    StreamThr :6132 [internal.py:wandb_internal():85] W&B internal server running at pid: 6132, started at: 2024-06-04 18:00:48.378871
2024-06-04 18:00:48,383 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: status
2024-06-04 18:00:48,384 INFO    WriterThread:6132 [datastore.py:open_for_write():87] open: /root/autodl-tmp/STllama/wandb/run-20240604_180048-gdhwwp6h/run-gdhwwp6h.wandb
2024-06-04 18:00:48,384 DEBUG   SenderThread:6132 [sender.py:send():378] send: header
2024-06-04 18:00:48,451 DEBUG   SenderThread:6132 [sender.py:send():378] send: run
2024-06-04 18:00:49,130 INFO    SenderThread:6132 [dir_watcher.py:__init__():211] watching files in: /root/autodl-tmp/STllama/wandb/run-20240604_180048-gdhwwp6h/files
2024-06-04 18:00:49,130 INFO    SenderThread:6132 [sender.py:_start_run_threads():1123] run started: gdhwwp6h with start time 1717495248.37877
2024-06-04 18:00:49,137 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: check_version
2024-06-04 18:00:49,137 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: check_version
2024-06-04 18:00:49,268 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: run_start
2024-06-04 18:00:49,279 DEBUG   HandlerThread:6132 [system_info.py:__init__():26] System info init
2024-06-04 18:00:49,279 DEBUG   HandlerThread:6132 [system_info.py:__init__():41] System info init done
2024-06-04 18:00:49,279 INFO    HandlerThread:6132 [system_monitor.py:start():194] Starting system monitor
2024-06-04 18:00:49,279 INFO    SystemMonitor:6132 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-06-04 18:00:49,280 INFO    HandlerThread:6132 [system_monitor.py:probe():214] Collecting system info
2024-06-04 18:00:49,281 INFO    SystemMonitor:6132 [interfaces.py:start():188] Started cpu monitoring
2024-06-04 18:00:49,282 INFO    SystemMonitor:6132 [interfaces.py:start():188] Started disk monitoring
2024-06-04 18:00:49,282 INFO    SystemMonitor:6132 [interfaces.py:start():188] Started gpu monitoring
2024-06-04 18:00:49,283 INFO    SystemMonitor:6132 [interfaces.py:start():188] Started memory monitoring
2024-06-04 18:00:49,286 INFO    SystemMonitor:6132 [interfaces.py:start():188] Started network monitoring
2024-06-04 18:00:49,302 DEBUG   HandlerThread:6132 [system_info.py:probe():150] Probing system
2024-06-04 18:00:49,309 DEBUG   HandlerThread:6132 [system_info.py:_probe_git():135] Probing git
2024-06-04 18:00:49,319 DEBUG   HandlerThread:6132 [system_info.py:_probe_git():143] Probing git done
2024-06-04 18:00:49,319 DEBUG   HandlerThread:6132 [system_info.py:probe():198] Probing system done
2024-06-04 18:00:49,319 DEBUG   HandlerThread:6132 [system_monitor.py:probe():223] {'os': 'Linux-5.4.0-99-generic-x86_64-with-glibc2.17', 'python': '3.8.10', 'heartbeatAt': '2024-06-04T10:00:49.302948', 'startedAt': '2024-06-04T10:00:48.373694', 'docker': None, 'cuda': None, 'args': ('--model_name_or_path', './checkpoints/vicuna-7b-v1.5-16k', '--version', 'v1', '--data_path', './data/discharge/st_gpt.json', '--st_content', './TAXI.json', '--st_data_path', './data/discharge/st_data.pkl', '--st_tower', 'ST_Encoder', '--tune_st_mlp_adapter', 'True', '--st_select_layer', '-2', '--use_st_start_end', '--fp16', 'True', '--output_dir', './checkpoints/UrbanGPT', '--num_train_epochs', '3', '--per_device_train_batch_size', '4', '--per_device_eval_batch_size', '4', '--gradient_accumulation_steps', '4', '--evaluation_strategy', 'no', '--save_strategy', 'steps', '--save_steps', '4800', '--save_total_limit', '1', '--learning_rate', '2e-3', '--weight_decay', '0.', '--warmup_ratio', '0.03', '--lr_scheduler_type', 'cosine', '--logging_steps', '1', '--tf32', 'False', '--model_max_length', '2048', '--gradient_checkpointing', 'True', '--lazy_preprocess', 'True', '--report_to', 'wandb'), 'state': 'running', 'program': 'urbangpt/train/train_mem.py', 'codePathLocal': 'urbangpt/train/train_mem.py', 'codePath': 'urbangpt/train/train_mem.py', 'git': {'remote': 'https://github.com/TobyYang7/STllama.git', 'commit': '8448b7de864dfbd0a2e72588f50828d5093a4b50'}, 'email': 'tobyyang7@foxmail.com', 'root': '/root/autodl-tmp/STllama', 'host': 'autodl-container-7bda11a2fa-d300d1c6', 'username': 'root', 'executable': '/root/miniconda3/bin/python', 'cpu_count': 40, 'cpu_count_logical': 80, 'cpu_freq': {'current': 3000.0279999999975, 'min': 0.0, 'max': 0.0}, 'cpu_freq_per_core': [{'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}, {'current': 3000.028, 'min': 0.0, 'max': 0.0}], 'disk': {'/': {'total': 30.0, 'used': 20.303916931152344}}, 'gpu': 'Tesla V100S-PCIE-32GB', 'gpu_count': 1, 'gpu_devices': [{'name': 'Tesla V100S-PCIE-32GB', 'memory_total': 34359738368}], 'memory': {'total': 629.8885612487793}}
2024-06-04 18:00:49,319 INFO    HandlerThread:6132 [system_monitor.py:probe():224] Finished collecting system info
2024-06-04 18:00:49,319 INFO    HandlerThread:6132 [system_monitor.py:probe():227] Publishing system info
2024-06-04 18:00:49,320 DEBUG   HandlerThread:6132 [system_info.py:_save_conda():207] Saving list of conda packages installed into the current environment
2024-06-04 18:00:49,838 DEBUG   HandlerThread:6132 [system_info.py:_save_conda():222] Saving conda packages done
2024-06-04 18:00:49,839 INFO    HandlerThread:6132 [system_monitor.py:probe():229] Finished publishing system info
2024-06-04 18:00:49,845 DEBUG   SenderThread:6132 [sender.py:send():378] send: files
2024-06-04 18:00:49,845 INFO    SenderThread:6132 [sender.py:_save_file():1389] saving file wandb-metadata.json with policy now
2024-06-04 18:00:49,999 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: python_packages
2024-06-04 18:00:50,000 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: python_packages
2024-06-04 18:00:50,000 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: stop_status
2024-06-04 18:00:50,001 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: stop_status
2024-06-04 18:00:50,131 INFO    Thread-12 :6132 [dir_watcher.py:_on_file_created():271] file/dir created: /root/autodl-tmp/STllama/wandb/run-20240604_180048-gdhwwp6h/files/conda-environment.yaml
2024-06-04 18:00:50,131 INFO    Thread-12 :6132 [dir_watcher.py:_on_file_created():271] file/dir created: /root/autodl-tmp/STllama/wandb/run-20240604_180048-gdhwwp6h/files/requirements.txt
2024-06-04 18:00:50,131 INFO    Thread-12 :6132 [dir_watcher.py:_on_file_created():271] file/dir created: /root/autodl-tmp/STllama/wandb/run-20240604_180048-gdhwwp6h/files/wandb-metadata.json
2024-06-04 18:00:50,434 DEBUG   SenderThread:6132 [sender.py:send():378] send: telemetry
2024-06-04 18:00:50,434 DEBUG   SenderThread:6132 [sender.py:send():378] send: config
2024-06-04 18:00:50,435 DEBUG   SenderThread:6132 [sender.py:send():378] send: metric
2024-06-04 18:00:50,435 DEBUG   SenderThread:6132 [sender.py:send():378] send: telemetry
2024-06-04 18:00:50,435 DEBUG   SenderThread:6132 [sender.py:send():378] send: telemetry
2024-06-04 18:00:50,435 DEBUG   SenderThread:6132 [sender.py:send():378] send: metric
2024-06-04 18:00:50,435 WARNING SenderThread:6132 [sender.py:send_metric():1340] Seen metric with glob (shouldn't happen)
2024-06-04 18:00:50,436 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/toby-yang/huggingface/gdhwwp6h/wandb-metadata.json?Expires=1717581650&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IP3ZBRJYK23vRgCKBIembGx%2Bjgxg5%2BPtS6bAg0jYgTmMFKCbwwiS%2F1cz2BeuJbGJ0NSNeUQ5hFE9qHYFLDPPoH3wAtn1F6CStY5qeAPjAEkSO5cXllF%2FAZaj1y1ASQaEbrpJ0SyH1FdFM96Pec0iy3IN4jt2NPRR7IMxuMrYp%2FpaMVu%2BTtkFYZImDlkfnELtpm%2FFZCZRaw0WPYLl6zNRsfNSvn9gvv0rRY3S2KhpHqW%2BPk17tiFwmEDh4xR%2B%2B8yLjqONOF0sGgRFNED6fR7cv5%2FzdJUEJXM3R1shB88L0kH0tYJEdLoMRBQ6xyjQvTr8P60IFEuR9gQm7%2FYHsewVYQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:00:50,436 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10734'}
2024-06-04 18:00:50,436 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:00:51,130 INFO    Thread-12 :6132 [dir_watcher.py:_on_file_created():271] file/dir created: /root/autodl-tmp/STllama/wandb/run-20240604_180048-gdhwwp6h/files/output.log
2024-06-04 18:00:51,265 DEBUG   SenderThread:6132 [sender.py:send():378] send: exit
2024-06-04 18:00:51,265 INFO    SenderThread:6132 [sender.py:send_exit():585] handling exit code: 1
2024-06-04 18:00:51,265 INFO    SenderThread:6132 [sender.py:send_exit():587] handling runtime: 1
2024-06-04 18:00:51,267 INFO    SenderThread:6132 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-06-04 18:00:51,267 INFO    SenderThread:6132 [sender.py:send_exit():593] send defer
2024-06-04 18:00:51,268 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: defer
2024-06-04 18:00:51,268 INFO    HandlerThread:6132 [handler.py:handle_request_defer():184] handle defer: 0
2024-06-04 18:00:51,268 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: defer
2024-06-04 18:00:51,268 INFO    SenderThread:6132 [sender.py:send_request_defer():609] handle sender defer: 0
2024-06-04 18:00:51,268 INFO    SenderThread:6132 [sender.py:transition_state():613] send defer: 1
2024-06-04 18:00:51,268 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: defer
2024-06-04 18:00:51,268 INFO    HandlerThread:6132 [handler.py:handle_request_defer():184] handle defer: 1
2024-06-04 18:00:51,269 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: defer
2024-06-04 18:00:51,269 INFO    SenderThread:6132 [sender.py:send_request_defer():609] handle sender defer: 1
2024-06-04 18:00:51,269 INFO    SenderThread:6132 [sender.py:transition_state():613] send defer: 2
2024-06-04 18:00:51,269 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: defer
2024-06-04 18:00:51,269 INFO    HandlerThread:6132 [handler.py:handle_request_defer():184] handle defer: 2
2024-06-04 18:00:51,269 INFO    HandlerThread:6132 [system_monitor.py:finish():203] Stopping system monitor
2024-06-04 18:00:51,269 DEBUG   SystemMonitor:6132 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-06-04 18:00:51,270 DEBUG   SystemMonitor:6132 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-06-04 18:00:51,270 DEBUG   SystemMonitor:6132 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-06-04 18:00:51,270 INFO    HandlerThread:6132 [interfaces.py:finish():200] Joined cpu monitor
2024-06-04 18:00:51,271 INFO    HandlerThread:6132 [interfaces.py:finish():200] Joined disk monitor
2024-06-04 18:00:51,296 INFO    HandlerThread:6132 [interfaces.py:finish():200] Joined gpu monitor
2024-06-04 18:00:51,296 INFO    HandlerThread:6132 [interfaces.py:finish():200] Joined memory monitor
2024-06-04 18:00:51,296 INFO    HandlerThread:6132 [interfaces.py:finish():200] Joined network monitor
2024-06-04 18:00:51,296 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: defer
2024-06-04 18:00:51,297 INFO    SenderThread:6132 [sender.py:send_request_defer():609] handle sender defer: 2
2024-06-04 18:00:51,297 INFO    SenderThread:6132 [sender.py:transition_state():613] send defer: 3
2024-06-04 18:00:51,297 DEBUG   SenderThread:6132 [sender.py:send():378] send: stats
2024-06-04 18:00:51,297 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: defer
2024-06-04 18:00:51,297 INFO    HandlerThread:6132 [handler.py:handle_request_defer():184] handle defer: 3
2024-06-04 18:00:51,297 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: defer
2024-06-04 18:00:51,297 INFO    SenderThread:6132 [sender.py:send_request_defer():609] handle sender defer: 3
2024-06-04 18:00:51,297 INFO    SenderThread:6132 [sender.py:transition_state():613] send defer: 4
2024-06-04 18:00:51,298 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: defer
2024-06-04 18:00:51,298 INFO    HandlerThread:6132 [handler.py:handle_request_defer():184] handle defer: 4
2024-06-04 18:00:51,298 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: defer
2024-06-04 18:00:51,298 INFO    SenderThread:6132 [sender.py:send_request_defer():609] handle sender defer: 4
2024-06-04 18:00:51,298 INFO    SenderThread:6132 [sender.py:transition_state():613] send defer: 5
2024-06-04 18:00:51,298 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: defer
2024-06-04 18:00:51,298 INFO    HandlerThread:6132 [handler.py:handle_request_defer():184] handle defer: 5
2024-06-04 18:00:51,298 DEBUG   SenderThread:6132 [sender.py:send():378] send: summary
2024-06-04 18:00:51,300 INFO    SenderThread:6132 [sender.py:_save_file():1389] saving file wandb-summary.json with policy end
2024-06-04 18:00:51,300 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: defer
2024-06-04 18:00:51,300 INFO    SenderThread:6132 [sender.py:send_request_defer():609] handle sender defer: 5
2024-06-04 18:00:51,300 INFO    SenderThread:6132 [sender.py:transition_state():613] send defer: 6
2024-06-04 18:00:51,301 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: defer
2024-06-04 18:00:51,301 INFO    HandlerThread:6132 [handler.py:handle_request_defer():184] handle defer: 6
2024-06-04 18:00:51,301 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: defer
2024-06-04 18:00:51,301 INFO    SenderThread:6132 [sender.py:send_request_defer():609] handle sender defer: 6
2024-06-04 18:00:51,305 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: status_report
2024-06-04 18:00:51,626 INFO    SenderThread:6132 [sender.py:transition_state():613] send defer: 7
2024-06-04 18:00:51,626 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: defer
2024-06-04 18:00:51,626 INFO    HandlerThread:6132 [handler.py:handle_request_defer():184] handle defer: 7
2024-06-04 18:00:51,627 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: defer
2024-06-04 18:00:51,627 INFO    SenderThread:6132 [sender.py:send_request_defer():609] handle sender defer: 7
2024-06-04 18:00:51,908 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/toby-yang/huggingface/gdhwwp6h/wandb-metadata.json?Expires=1717581650&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IP3ZBRJYK23vRgCKBIembGx%2Bjgxg5%2BPtS6bAg0jYgTmMFKCbwwiS%2F1cz2BeuJbGJ0NSNeUQ5hFE9qHYFLDPPoH3wAtn1F6CStY5qeAPjAEkSO5cXllF%2FAZaj1y1ASQaEbrpJ0SyH1FdFM96Pec0iy3IN4jt2NPRR7IMxuMrYp%2FpaMVu%2BTtkFYZImDlkfnELtpm%2FFZCZRaw0WPYLl6zNRsfNSvn9gvv0rRY3S2KhpHqW%2BPk17tiFwmEDh4xR%2B%2B8yLjqONOF0sGgRFNED6fR7cv5%2FzdJUEJXM3R1shB88L0kH0tYJEdLoMRBQ6xyjQvTr8P60IFEuR9gQm7%2FYHsewVYQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:00:51,908 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10734'}
2024-06-04 18:00:51,908 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:00:52,131 INFO    Thread-12 :6132 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/autodl-tmp/STllama/wandb/run-20240604_180048-gdhwwp6h/files/config.yaml
2024-06-04 18:00:52,131 INFO    Thread-12 :6132 [dir_watcher.py:_on_file_created():271] file/dir created: /root/autodl-tmp/STllama/wandb/run-20240604_180048-gdhwwp6h/files/wandb-summary.json
2024-06-04 18:00:52,265 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: poll_exit
2024-06-04 18:00:52,447 INFO    SenderThread:6132 [sender.py:transition_state():613] send defer: 8
2024-06-04 18:00:52,447 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: poll_exit
2024-06-04 18:00:52,448 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: defer
2024-06-04 18:00:52,448 INFO    HandlerThread:6132 [handler.py:handle_request_defer():184] handle defer: 8
2024-06-04 18:00:52,448 DEBUG   SenderThread:6132 [sender.py:send_request():405] send_request: defer
2024-06-04 18:00:52,448 INFO    SenderThread:6132 [sender.py:send_request_defer():609] handle sender defer: 8
2024-06-04 18:00:52,449 INFO    SenderThread:6132 [job_builder.py:build():432] Attempting to build job artifact
2024-06-04 18:00:52,452 INFO    SenderThread:6132 [job_builder.py:_get_source_type():565] is repo sourced job
2024-06-04 18:00:52,482 INFO    SenderThread:6132 [job_builder.py:build():541] adding wandb-job metadata file
2024-06-04 18:00:52,488 INFO    SenderThread:6132 [sender.py:transition_state():613] send defer: 9
2024-06-04 18:00:52,488 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: defer
2024-06-04 18:00:52,488 INFO    HandlerThread:6132 [handler.py:handle_request_defer():184] handle defer: 9
2024-06-04 18:00:52,489 DEBUG   SenderThread:6132 [sender.py:send():378] send: artifact
2024-06-04 18:00:53,131 INFO    Thread-12 :6132 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/autodl-tmp/STllama/wandb/run-20240604_180048-gdhwwp6h/files/output.log
2024-06-04 18:00:53,265 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: poll_exit
2024-06-04 18:00:54,019 INFO    wandb-upload_1:6132 [upload_job.py:push():85] Skipped uploading /root/.local/share/wandb/artifacts/staging/tmpdroekmw7
2024-06-04 18:00:54,237 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/toby-yang/huggingface/gdhwwp6h/wandb-metadata.json?Expires=1717581650&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IP3ZBRJYK23vRgCKBIembGx%2Bjgxg5%2BPtS6bAg0jYgTmMFKCbwwiS%2F1cz2BeuJbGJ0NSNeUQ5hFE9qHYFLDPPoH3wAtn1F6CStY5qeAPjAEkSO5cXllF%2FAZaj1y1ASQaEbrpJ0SyH1FdFM96Pec0iy3IN4jt2NPRR7IMxuMrYp%2FpaMVu%2BTtkFYZImDlkfnELtpm%2FFZCZRaw0WPYLl6zNRsfNSvn9gvv0rRY3S2KhpHqW%2BPk17tiFwmEDh4xR%2B%2B8yLjqONOF0sGgRFNED6fR7cv5%2FzdJUEJXM3R1shB88L0kH0tYJEdLoMRBQ6xyjQvTr8P60IFEuR9gQm7%2FYHsewVYQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:00:54,238 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10734'}
2024-06-04 18:00:54,238 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:00:54,293 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/181782398/859757826/f14b3dbf2547b3e520fc5c05858910cd?Expires=1717581653&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ws5dRSsdL2cRNoKa5FthpoRSpg5%2Fv8mt0jRSZuL7TM5m2WZSZnHxfr%2FX4lLYFKs8wvJX5HLHFVGfpFcZ%2Bbo5bzb5LkIkk%2BCqPUXloPTvuxS05Fe9LfshhK7VXljcQQO%2BTljPTp%2F28JpXNa1t9vjFxZ6VW3ZLsIpUb%2Fj95nMxch8dza6xgOIQEmI7kFfrrLwcF7rvrhH2Lb9UA1dDq484o6ZLwu%2FBkVHlQbgjxIbxvdTj6z3QIxvs2zRT3hw5uMaLpyX8BhjBW18HVevYk0aQi13V4npSSHTKObYUWjmNXc0a%2FKAbDBgNDR5QcZXd2W2%2BnSfa%2Bomwiau7sIMEA4c2aQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:00:54,293 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '8Us9vyVHs+Ug/FwFhYkQzQ==', 'Content-Type': 'application/json', 'Content-Length': '23149'}
2024-06-04 18:00:54,293 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:00:55,559 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/181782398/859757826/f14b3dbf2547b3e520fc5c05858910cd?Expires=1717581653&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ws5dRSsdL2cRNoKa5FthpoRSpg5%2Fv8mt0jRSZuL7TM5m2WZSZnHxfr%2FX4lLYFKs8wvJX5HLHFVGfpFcZ%2Bbo5bzb5LkIkk%2BCqPUXloPTvuxS05Fe9LfshhK7VXljcQQO%2BTljPTp%2F28JpXNa1t9vjFxZ6VW3ZLsIpUb%2Fj95nMxch8dza6xgOIQEmI7kFfrrLwcF7rvrhH2Lb9UA1dDq484o6ZLwu%2FBkVHlQbgjxIbxvdTj6z3QIxvs2zRT3hw5uMaLpyX8BhjBW18HVevYk0aQi13V4npSSHTKObYUWjmNXc0a%2FKAbDBgNDR5QcZXd2W2%2BnSfa%2Bomwiau7sIMEA4c2aQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:00:55,559 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '8Us9vyVHs+Ug/FwFhYkQzQ==', 'Content-Type': 'application/json', 'Content-Length': '23149'}
2024-06-04 18:00:55,559 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:00:55,559 INFO    wandb-upload_2:6132 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 404, in _make_request
    self._validate_conn(conn)
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1058, in _validate_conn
    conn.connect()
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connection.py", line 419, in connect
    self.sock = ssl_wrap_socket(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/root/miniconda3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/root/miniconda3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/root/miniconda3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 489, in send
    resp = conn.urlopen(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 799, in urlopen
    retries = retries.increment(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/retry.py", line 550, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/packages/six.py", line 769, in reraise
    raise value.with_traceback(tb)
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 404, in _make_request
    self._validate_conn(conn)
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1058, in _validate_conn
    conn.connect()
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/connection.py", line 419, in connect
    self.sock = ssl_wrap_socket(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/root/miniconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/root/miniconda3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/root/miniconda3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/root/miniconda3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2731, in upload_file
    response = self._upload_file_session.put(
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 547, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2762, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/root/miniconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 2731, in upload_file
    response = self._upload_file_session.put(
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/root/miniconda3/lib/python3.8/site-packages/requests/adapters.py", line 547, in send
    raise ConnectionError(err, request=request)
wandb.sdk.lib.retry.TransientError: None
2024-06-04 18:00:58,168 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/181782398/859757826/f14b3dbf2547b3e520fc5c05858910cd?Expires=1717581653&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ws5dRSsdL2cRNoKa5FthpoRSpg5%2Fv8mt0jRSZuL7TM5m2WZSZnHxfr%2FX4lLYFKs8wvJX5HLHFVGfpFcZ%2Bbo5bzb5LkIkk%2BCqPUXloPTvuxS05Fe9LfshhK7VXljcQQO%2BTljPTp%2F28JpXNa1t9vjFxZ6VW3ZLsIpUb%2Fj95nMxch8dza6xgOIQEmI7kFfrrLwcF7rvrhH2Lb9UA1dDq484o6ZLwu%2FBkVHlQbgjxIbxvdTj6z3QIxvs2zRT3hw5uMaLpyX8BhjBW18HVevYk0aQi13V4npSSHTKObYUWjmNXc0a%2FKAbDBgNDR5QcZXd2W2%2BnSfa%2Bomwiau7sIMEA4c2aQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:00:58,168 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '8Us9vyVHs+Ug/FwFhYkQzQ==', 'Content-Type': 'application/json', 'Content-Length': '23149'}
2024-06-04 18:00:58,169 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:00:58,273 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:00:58,914 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/toby-yang/huggingface/gdhwwp6h/wandb-metadata.json?Expires=1717581650&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IP3ZBRJYK23vRgCKBIembGx%2Bjgxg5%2BPtS6bAg0jYgTmMFKCbwwiS%2F1cz2BeuJbGJ0NSNeUQ5hFE9qHYFLDPPoH3wAtn1F6CStY5qeAPjAEkSO5cXllF%2FAZaj1y1ASQaEbrpJ0SyH1FdFM96Pec0iy3IN4jt2NPRR7IMxuMrYp%2FpaMVu%2BTtkFYZImDlkfnELtpm%2FFZCZRaw0WPYLl6zNRsfNSvn9gvv0rRY3S2KhpHqW%2BPk17tiFwmEDh4xR%2B%2B8yLjqONOF0sGgRFNED6fR7cv5%2FzdJUEJXM3R1shB88L0kH0tYJEdLoMRBQ6xyjQvTr8P60IFEuR9gQm7%2FYHsewVYQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:00:58,914 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10734'}
2024-06-04 18:00:58,914 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:01:02,511 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/181782398/859757826/f14b3dbf2547b3e520fc5c05858910cd?Expires=1717581653&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ws5dRSsdL2cRNoKa5FthpoRSpg5%2Fv8mt0jRSZuL7TM5m2WZSZnHxfr%2FX4lLYFKs8wvJX5HLHFVGfpFcZ%2Bbo5bzb5LkIkk%2BCqPUXloPTvuxS05Fe9LfshhK7VXljcQQO%2BTljPTp%2F28JpXNa1t9vjFxZ6VW3ZLsIpUb%2Fj95nMxch8dza6xgOIQEmI7kFfrrLwcF7rvrhH2Lb9UA1dDq484o6ZLwu%2FBkVHlQbgjxIbxvdTj6z3QIxvs2zRT3hw5uMaLpyX8BhjBW18HVevYk0aQi13V4npSSHTKObYUWjmNXc0a%2FKAbDBgNDR5QcZXd2W2%2BnSfa%2Bomwiau7sIMEA4c2aQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:01:02,512 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '8Us9vyVHs+Ug/FwFhYkQzQ==', 'Content-Type': 'application/json', 'Content-Length': '23149'}
2024-06-04 18:01:02,512 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:01:03,275 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:07,205 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/toby-yang/huggingface/gdhwwp6h/wandb-metadata.json?Expires=1717581650&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IP3ZBRJYK23vRgCKBIembGx%2Bjgxg5%2BPtS6bAg0jYgTmMFKCbwwiS%2F1cz2BeuJbGJ0NSNeUQ5hFE9qHYFLDPPoH3wAtn1F6CStY5qeAPjAEkSO5cXllF%2FAZaj1y1ASQaEbrpJ0SyH1FdFM96Pec0iy3IN4jt2NPRR7IMxuMrYp%2FpaMVu%2BTtkFYZImDlkfnELtpm%2FFZCZRaw0WPYLl6zNRsfNSvn9gvv0rRY3S2KhpHqW%2BPk17tiFwmEDh4xR%2B%2B8yLjqONOF0sGgRFNED6fR7cv5%2FzdJUEJXM3R1shB88L0kH0tYJEdLoMRBQ6xyjQvTr8P60IFEuR9gQm7%2FYHsewVYQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:01:07,205 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10734'}
2024-06-04 18:01:07,205 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:01:08,277 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:12,055 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/181782398/859757826/f14b3dbf2547b3e520fc5c05858910cd?Expires=1717581653&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ws5dRSsdL2cRNoKa5FthpoRSpg5%2Fv8mt0jRSZuL7TM5m2WZSZnHxfr%2FX4lLYFKs8wvJX5HLHFVGfpFcZ%2Bbo5bzb5LkIkk%2BCqPUXloPTvuxS05Fe9LfshhK7VXljcQQO%2BTljPTp%2F28JpXNa1t9vjFxZ6VW3ZLsIpUb%2Fj95nMxch8dza6xgOIQEmI7kFfrrLwcF7rvrhH2Lb9UA1dDq484o6ZLwu%2FBkVHlQbgjxIbxvdTj6z3QIxvs2zRT3hw5uMaLpyX8BhjBW18HVevYk0aQi13V4npSSHTKObYUWjmNXc0a%2FKAbDBgNDR5QcZXd2W2%2BnSfa%2Bomwiau7sIMEA4c2aQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:01:12,056 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '8Us9vyVHs+Ug/FwFhYkQzQ==', 'Content-Type': 'application/json', 'Content-Length': '23149'}
2024-06-04 18:01:12,056 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:01:13,280 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:18,282 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:23,284 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:23,404 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/toby-yang/huggingface/gdhwwp6h/wandb-metadata.json?Expires=1717581650&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IP3ZBRJYK23vRgCKBIembGx%2Bjgxg5%2BPtS6bAg0jYgTmMFKCbwwiS%2F1cz2BeuJbGJ0NSNeUQ5hFE9qHYFLDPPoH3wAtn1F6CStY5qeAPjAEkSO5cXllF%2FAZaj1y1ASQaEbrpJ0SyH1FdFM96Pec0iy3IN4jt2NPRR7IMxuMrYp%2FpaMVu%2BTtkFYZImDlkfnELtpm%2FFZCZRaw0WPYLl6zNRsfNSvn9gvv0rRY3S2KhpHqW%2BPk17tiFwmEDh4xR%2B%2B8yLjqONOF0sGgRFNED6fR7cv5%2FzdJUEJXM3R1shB88L0kH0tYJEdLoMRBQ6xyjQvTr8P60IFEuR9gQm7%2FYHsewVYQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:01:23,404 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10734'}
2024-06-04 18:01:23,404 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:01:28,287 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:30,755 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/181782398/859757826/f14b3dbf2547b3e520fc5c05858910cd?Expires=1717581653&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ws5dRSsdL2cRNoKa5FthpoRSpg5%2Fv8mt0jRSZuL7TM5m2WZSZnHxfr%2FX4lLYFKs8wvJX5HLHFVGfpFcZ%2Bbo5bzb5LkIkk%2BCqPUXloPTvuxS05Fe9LfshhK7VXljcQQO%2BTljPTp%2F28JpXNa1t9vjFxZ6VW3ZLsIpUb%2Fj95nMxch8dza6xgOIQEmI7kFfrrLwcF7rvrhH2Lb9UA1dDq484o6ZLwu%2FBkVHlQbgjxIbxvdTj6z3QIxvs2zRT3hw5uMaLpyX8BhjBW18HVevYk0aQi13V4npSSHTKObYUWjmNXc0a%2FKAbDBgNDR5QcZXd2W2%2BnSfa%2Bomwiau7sIMEA4c2aQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:01:30,755 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '8Us9vyVHs+Ug/FwFhYkQzQ==', 'Content-Type': 'application/json', 'Content-Length': '23149'}
2024-06-04 18:01:30,755 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:01:33,289 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:38,292 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:43,294 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:48,296 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:53,299 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:58,301 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:01:59,247 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/toby-yang/huggingface/gdhwwp6h/wandb-metadata.json?Expires=1717581650&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IP3ZBRJYK23vRgCKBIembGx%2Bjgxg5%2BPtS6bAg0jYgTmMFKCbwwiS%2F1cz2BeuJbGJ0NSNeUQ5hFE9qHYFLDPPoH3wAtn1F6CStY5qeAPjAEkSO5cXllF%2FAZaj1y1ASQaEbrpJ0SyH1FdFM96Pec0iy3IN4jt2NPRR7IMxuMrYp%2FpaMVu%2BTtkFYZImDlkfnELtpm%2FFZCZRaw0WPYLl6zNRsfNSvn9gvv0rRY3S2KhpHqW%2BPk17tiFwmEDh4xR%2B%2B8yLjqONOF0sGgRFNED6fR7cv5%2FzdJUEJXM3R1shB88L0kH0tYJEdLoMRBQ6xyjQvTr8P60IFEuR9gQm7%2FYHsewVYQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:01:59,247 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10734'}
2024-06-04 18:01:59,247 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:02:03,304 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:02:08,306 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:02:10,134 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/181782398/859757826/f14b3dbf2547b3e520fc5c05858910cd?Expires=1717581653&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ws5dRSsdL2cRNoKa5FthpoRSpg5%2Fv8mt0jRSZuL7TM5m2WZSZnHxfr%2FX4lLYFKs8wvJX5HLHFVGfpFcZ%2Bbo5bzb5LkIkk%2BCqPUXloPTvuxS05Fe9LfshhK7VXljcQQO%2BTljPTp%2F28JpXNa1t9vjFxZ6VW3ZLsIpUb%2Fj95nMxch8dza6xgOIQEmI7kFfrrLwcF7rvrhH2Lb9UA1dDq484o6ZLwu%2FBkVHlQbgjxIbxvdTj6z3QIxvs2zRT3hw5uMaLpyX8BhjBW18HVevYk0aQi13V4npSSHTKObYUWjmNXc0a%2FKAbDBgNDR5QcZXd2W2%2BnSfa%2Bomwiau7sIMEA4c2aQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:02:10,135 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '8Us9vyVHs+Ug/FwFhYkQzQ==', 'Content-Type': 'application/json', 'Content-Length': '23149'}
2024-06-04 18:02:10,135 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:02:13,309 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:02:18,312 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:02:23,315 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:02:28,318 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:02:33,320 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:02:38,322 DEBUG   HandlerThread:6132 [handler.py:handle_request():158] handle_request: keepalive
2024-06-04 18:02:41,509 WARNING StreamThr :6132 [internal.py:is_dead():413] Internal process exiting, parent pid 6066 disappeared
2024-06-04 18:02:41,509 ERROR   StreamThr :6132 [internal.py:wandb_internal():151] Internal process shutdown.
2024-06-04 18:02:42,290 INFO    WriterThread:6132 [datastore.py:close():296] close: /root/autodl-tmp/STllama/wandb/run-20240604_180048-gdhwwp6h/run-gdhwwp6h.wandb
2024-06-04 18:02:42,323 INFO    HandlerThread:6132 [handler.py:finish():882] shutting down handler
2024-06-04 18:03:12,496 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/toby-yang/huggingface/gdhwwp6h/wandb-metadata.json?Expires=1717581650&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IP3ZBRJYK23vRgCKBIembGx%2Bjgxg5%2BPtS6bAg0jYgTmMFKCbwwiS%2F1cz2BeuJbGJ0NSNeUQ5hFE9qHYFLDPPoH3wAtn1F6CStY5qeAPjAEkSO5cXllF%2FAZaj1y1ASQaEbrpJ0SyH1FdFM96Pec0iy3IN4jt2NPRR7IMxuMrYp%2FpaMVu%2BTtkFYZImDlkfnELtpm%2FFZCZRaw0WPYLl6zNRsfNSvn9gvv0rRY3S2KhpHqW%2BPk17tiFwmEDh4xR%2B%2B8yLjqONOF0sGgRFNED6fR7cv5%2FzdJUEJXM3R1shB88L0kH0tYJEdLoMRBQ6xyjQvTr8P60IFEuR9gQm7%2FYHsewVYQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:03:12,496 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10734'}
2024-06-04 18:03:12,496 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:03:30,471 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/181782398/859757826/f14b3dbf2547b3e520fc5c05858910cd?Expires=1717581653&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ws5dRSsdL2cRNoKa5FthpoRSpg5%2Fv8mt0jRSZuL7TM5m2WZSZnHxfr%2FX4lLYFKs8wvJX5HLHFVGfpFcZ%2Bbo5bzb5LkIkk%2BCqPUXloPTvuxS05Fe9LfshhK7VXljcQQO%2BTljPTp%2F28JpXNa1t9vjFxZ6VW3ZLsIpUb%2Fj95nMxch8dza6xgOIQEmI7kFfrrLwcF7rvrhH2Lb9UA1dDq484o6ZLwu%2FBkVHlQbgjxIbxvdTj6z3QIxvs2zRT3hw5uMaLpyX8BhjBW18HVevYk0aQi13V4npSSHTKObYUWjmNXc0a%2FKAbDBgNDR5QcZXd2W2%2BnSfa%2Bomwiau7sIMEA4c2aQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:03:30,471 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '8Us9vyVHs+Ug/FwFhYkQzQ==', 'Content-Type': 'application/json', 'Content-Length': '23149'}
2024-06-04 18:03:30,471 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:05:32,388 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/toby-yang/huggingface/gdhwwp6h/wandb-metadata.json?Expires=1717581650&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IP3ZBRJYK23vRgCKBIembGx%2Bjgxg5%2BPtS6bAg0jYgTmMFKCbwwiS%2F1cz2BeuJbGJ0NSNeUQ5hFE9qHYFLDPPoH3wAtn1F6CStY5qeAPjAEkSO5cXllF%2FAZaj1y1ASQaEbrpJ0SyH1FdFM96Pec0iy3IN4jt2NPRR7IMxuMrYp%2FpaMVu%2BTtkFYZImDlkfnELtpm%2FFZCZRaw0WPYLl6zNRsfNSvn9gvv0rRY3S2KhpHqW%2BPk17tiFwmEDh4xR%2B%2B8yLjqONOF0sGgRFNED6fR7cv5%2FzdJUEJXM3R1shB88L0kH0tYJEdLoMRBQ6xyjQvTr8P60IFEuR9gQm7%2FYHsewVYQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:05:32,388 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10734'}
2024-06-04 18:05:32,388 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:05:56,888 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/181782398/859757826/f14b3dbf2547b3e520fc5c05858910cd?Expires=1717581653&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ws5dRSsdL2cRNoKa5FthpoRSpg5%2Fv8mt0jRSZuL7TM5m2WZSZnHxfr%2FX4lLYFKs8wvJX5HLHFVGfpFcZ%2Bbo5bzb5LkIkk%2BCqPUXloPTvuxS05Fe9LfshhK7VXljcQQO%2BTljPTp%2F28JpXNa1t9vjFxZ6VW3ZLsIpUb%2Fj95nMxch8dza6xgOIQEmI7kFfrrLwcF7rvrhH2Lb9UA1dDq484o6ZLwu%2FBkVHlQbgjxIbxvdTj6z3QIxvs2zRT3hw5uMaLpyX8BhjBW18HVevYk0aQi13V4npSSHTKObYUWjmNXc0a%2FKAbDBgNDR5QcZXd2W2%2BnSfa%2Bomwiau7sIMEA4c2aQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:05:56,889 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '8Us9vyVHs+Ug/FwFhYkQzQ==', 'Content-Type': 'application/json', 'Content-Length': '23149'}
2024-06-04 18:05:56,889 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:09:55,416 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/toby-yang/huggingface/gdhwwp6h/wandb-metadata.json?Expires=1717581650&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=IP3ZBRJYK23vRgCKBIembGx%2Bjgxg5%2BPtS6bAg0jYgTmMFKCbwwiS%2F1cz2BeuJbGJ0NSNeUQ5hFE9qHYFLDPPoH3wAtn1F6CStY5qeAPjAEkSO5cXllF%2FAZaj1y1ASQaEbrpJ0SyH1FdFM96Pec0iy3IN4jt2NPRR7IMxuMrYp%2FpaMVu%2BTtkFYZImDlkfnELtpm%2FFZCZRaw0WPYLl6zNRsfNSvn9gvv0rRY3S2KhpHqW%2BPk17tiFwmEDh4xR%2B%2B8yLjqONOF0sGgRFNED6fR7cv5%2FzdJUEJXM3R1shB88L0kH0tYJEdLoMRBQ6xyjQvTr8P60IFEuR9gQm7%2FYHsewVYQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:09:55,416 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10734'}
2024-06-04 18:09:55,416 ERROR   wandb-upload_0:6132 [internal_api.py:upload_file():2742] upload_file response body: 
2024-06-04 18:10:25,437 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2738] upload_file exception https://storage.googleapis.com/wandb-artifacts-prod/wandb_artifacts/181782398/859757826/f14b3dbf2547b3e520fc5c05858910cd?Expires=1717581653&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ws5dRSsdL2cRNoKa5FthpoRSpg5%2Fv8mt0jRSZuL7TM5m2WZSZnHxfr%2FX4lLYFKs8wvJX5HLHFVGfpFcZ%2Bbo5bzb5LkIkk%2BCqPUXloPTvuxS05Fe9LfshhK7VXljcQQO%2BTljPTp%2F28JpXNa1t9vjFxZ6VW3ZLsIpUb%2Fj95nMxch8dza6xgOIQEmI7kFfrrLwcF7rvrhH2Lb9UA1dDq484o6ZLwu%2FBkVHlQbgjxIbxvdTj6z3QIxvs2zRT3hw5uMaLpyX8BhjBW18HVevYk0aQi13V4npSSHTKObYUWjmNXc0a%2FKAbDBgNDR5QcZXd2W2%2BnSfa%2Bomwiau7sIMEA4c2aQ%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-06-04 18:10:25,438 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2740] upload_file request headers: {'User-Agent': 'python-requests/2.28.2', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-MD5': '8Us9vyVHs+Ug/FwFhYkQzQ==', 'Content-Type': 'application/json', 'Content-Length': '23149'}
2024-06-04 18:10:25,438 ERROR   wandb-upload_2:6132 [internal_api.py:upload_file():2742] upload_file response body: 
